fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/BackendsAction.java

```diff
@@ -70,9 +67,6 @@ public class BackendsAction extends RestBaseController {
     @RequestMapping(path = "/api/backends", method = {RequestMethod.GET})
     public Object getBackends(HttpServletRequest request, HttpServletResponse response) {
         executeCheckPassword(request, response);
-        if (Config.enable_all_http_auth) {
```

Comment:
add some comments to explain why this api without auth

Reply:
I've explained that in the issue.

Reply:
Do I need to explain that in the code？